test/services/database/user_repository_test.dart

```diff
@@ -140,5 +140,41 @@ void main() {
         expectedUser.data.centauriPoints + pointsToAdd,
       );
     });
+
+    test("User ranking by top centauri points with enough users", () async {
+      const numberOfUsers = 10;
+      const limit = 5;
+
+      // Generate users with different centauri points
+      await FirestoreUserGenerator.addUsers(fakeFireStore, numberOfUsers);
+
+      // Query top users using the service under test (actual)
+      final topUsers = await userRepo.getTopUsers(limit);
+
+      // Check that correct number of users are returned
+      expect(topUsers.length, limit);
```

Comment:
Applied in 86c5a20 (nitpick)

Reply:
I made this comment since Yoann made it on some of my PRs. I communicated to the rest of the team and we will try to avoid this kind of review comment in the future reviews, thank you! :)